devel/ansible/roles/bodhi/tasks/rabbitmq.yml

```diff
@@ -24,6 +24,14 @@
   notify:
     - reload rabbitmq
 
+- name: Make sure /var/lib/rabbitmq is owned by the rabbitmq user
```

Comment:
Is this tied to a bug in the F30 rabbitmq package? If so, it would be nice to add a reference to that bug so we can remove those lines when it's fixed.

Reply:
> Is this tied to a bug in the F30 rabbitmq package? If so, it would be nice to add a reference to that bug so we can remove those lines when it's fixed.

To be honest I have not investigated further than that, the rabbitmq-server service was failing because of the wrong permission on this directory. I ll check if there is a bug for that in BZ

Reply:
So for some reason `/var/lib/rabbitmq/.erlang.cookie` is created and owned by root before we try to start the service. I ll try to investigate that further